serialization.md

```diff
@@ -327,7 +327,6 @@ Note: whitespace has been added to this example for clarity. Whitespace is OPTIO
         <ul>
           <li>
             <code>type</code> is usually set to <code>layers</code>.
-	    There is also a Windows-specific value <code>layers+base</code> that allows a base layer to be specified in a field of <code>rootfs</code> called <code>base_layer</code>.
```

Comment:
On Mon, Aug 29, 2016 at 05:41:40PM -0700, Stephen Day wrote:
> Removing this field probably has unintended consequences.

“Unintended consequences” are hard to discuss ;).  Can you float an
example of the sort of consequences you're concerned about?

> Recently, this value for the field was removed from the docker
> landscape due to a better solution for identifying base image layers
> that could not be distributed. The field value would never have been
> seen in production ready versions of docker and has no place in OCI.

Yeah, I think “we want to remove layers+base” is something that we
agree on.

> The scope of this PR was to remove the value for the field and bring
> it in line with real world usage. Any discussion around changes to
> the meaning of the field or whether the field exists at all are
> outside the scope of this PR.
>
> Keeping the discussion focused will ensure that PRs move more
> quickly and don't end up in a state where the path forward is
> unclear.

Touching/removing a particular line seems like a good opportunity to
consider the sense of the surrounding material with less maintainer
context-switching.  But if getting to the bottom of the rootfs.type
field as a whole is too much to bite off in this PR, I'm happy to punt
everything after dropping layers+base to follow-up PRs.  It's possible
that a general increase in formality like that proposed in #220 will
also help clarify rootfs.type as well.